articles/credits-faq.md

```diff
@@ -0,0 +1,107 @@
+---
+author: mblouin
+description: Frequently Asked Questions regarding the Azure Quantum Credits Program
+ms.author: mblouin
+ms.date: 10/06/2021
+ms.service: azure-quantum
+ms.subservice: credits
+ms.topic: faq
+title: Azure Quantum Credits FAQ
+uid: microsoft.quantum.credits.credits-faq
+---
+
+# FAQ: Applications to the Azure Quantum Credits Program
+
+## I have questions about Azure Quantum or Azure Quantum Credits and I can't find the answer, how do I get help?
+If you have an Azure support subscription, we recommend connecting with Azure Support for the fastest resolution to your question.
+
+If you have any question regarding the workspace creation process or run into any issue at any point using Azure Quantum, bookmark [https://aka.ms/AQ/OfficeHours](https://aka.ms/AQ/OfficeHours) and join our open office hours every Thursday at 8:30am PST.
+
+## Who can apply for the Azure Quantum Credits Program?
+Educational institutions, companies, and some other approved organizations may apply for Azure Quantum Credits. Credits applicants may be based in the United States, or in any other market that is currently served by Microsoft Azure. For more information, see the [Azure Quantum Hardware Credit Program Terms](https://aka.ms/aq/credits/terms).
+
+## What are the terms of use for Azure Quantum Credits?
+You can find the terms for the credits program terms at this link: [aka.ms/aq/credits/terms](https://aka.ms/aq/credits/terms).
+
+Note that in using Azure Quantum Credits you must also accept the relevant usage terms for Microsoft Azure, and the terms from the providers you would like access to.
+
+## How long are credits valid for?
+Credits are valid 6 months from your project start date. Extensions are occasionally available on a case-by-case basis.
+
+## I have applied, how long will it take to hear back on my application?
+We try to respond to all credits applications within one (1) month of the application date.
+
+## What programming languages or frameworks can be used with Azure Quantum and Azure Quantum Credits?
+Credits may be used with any programming language or framework that is supported by Azure Quantum. Currently we support:
+
+- Q# ([Quickstart](xref:microsoft.quantum.quickstarts.computing)) ([Installation](https://docs.microsoft.com/azure/quantum/quickstart-microsoft-qc))
+- Cirq ([Quickstart](xref:microsoft.quantum.quickstarts.computing.cirq)) ([Installation](https://quantumai.google/cirq/azure-quantum/access))
+- Qiskit ([Quickstart](xref:microsoft.quantum.quickstarts.computing.qiskit)) ([Installation]())
```

Comment:
This is confusing - 

- For Q#, the quickstart link and the install link are the same.
- For Cirq, the Google install page just has our quickstart instructions repeated. 
- Not sure about Qiskit installation...

Can we just link to our quickstarts?

Reply:
Yep, I think I'll just link the quickstarts